code/modules/mob/dead/observer/observer.dm

```diff
@@ -211,8 +200,6 @@ GLOBAL_VAR_INIT(observer_default_invisibility, INVISIBILITY_OBSERVER)
 
 		switch(tgui_alert(ghost, "What would you like to do?", "Burrowed larva source available", list("Join as Larva", "Cancel"), 0))
 			if("Join as Larva")
-				can_reenter_corpse = null
```

Comment:
why'd you remove this entirely?

Reply:
I'm not sure that it's needed for making them show as DNR (flatline hud icon), I will proabbly add back setting their corpse as undefibbable though